roofit/roofitcore/src/RooDataHist.cxx

```diff
@@ -1474,7 +1474,7 @@ void RooDataHist::set(std::size_t binNumber, double wgt, double wgtErr) {
 /// \deprecated Prefer set(std::size_t, double, double).
 /// \param[in] wgt New bin content.
 /// \param[in] wgtErr Optional error of the bin content.
-void RooDataHist::set(double wgt, double wgtErr) {
+void RooDataHist::set(Double_t wgt, Double_t wgtErr) {
```

Comment:
I would remove that. RooFit is trying to go more in the direction of standard c++, but here you are going back.

Reply:
I agree, but then the header file should be also with double instead of Double_t, no?

Reply:
Yes! 🙂
But if you do this, please do it in a separate commit.

Reply:
I rebased to another branch https://github.com/root-project/root/pull/9657, closing this.